sdk/identity/identity/src/credentials/deviceCodeCredential.ts

```diff
@@ -131,12 +133,10 @@ export class DeviceCodeCredential implements TokenCredential {
 
       if (err.name === AuthenticationErrorName) {
         logger.warning(
-          `DeviceCodeCredential: failed to authenticate ${
-            (err as AuthenticationError).errorResponse.errorDescription
-          }`
+          `Failed to authenticate ${(err as AuthenticationError).errorResponse.errorDescription}`
         );
       } else {
-        logger.warning(`DeviceCodeCredential: failed to authenticate ${err}`);
+        logger.warning(`Failed to authenticate ${err}`);
```

Comment:
Just checking, is the warning level where all you errors are reported? Shouldn't this also be at the error level?

Reply:
That is a great question. Thank you for bringing attention to this! I will change these instances to `error`.